cmd/login/wc.go

```diff
@@ -214,5 +296,24 @@ func getClusterBasePath(k8sConfigAccess clientcmd.ConfigAccess) (string, error)
 	// Some management clusters might have 'api.g8s' as prefix (example: Viking).
 	clusterServer = strings.TrimPrefix(clusterServer, "https://api.g8s.")
 
+	// openstack clusters have an api.$INSTALLATION prefix
+	if provider == key.ProviderOpenStack {
```

Comment:
I still don't like this, but I don't think we can fix it in this PR. I have this WIP PR to improve the situation https://github.com/giantswarm/kubectl-gs/pull/713